csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -562,6 +561,39 @@ List of URLs (`urls`) of acknowledgment is a container (value type `array`) for
 Any URL of acknowledgment contains the URL or location of the reference to be acknowledged.
 Value type is string with format URI (`uri`).
 
+##### 3.1.1.5 Acknowledgments Type - Example
+
```

Comment:
According to the rules in the document this should be an informative section. Therefore, we should add before the example the following:
```suggestion

Example:

```